mina.netty/src/main/java/org/jboss/netty/channel/socket/nio/NioDatagramChannel.java

```diff
@@ -70,7 +70,7 @@
 
     NioDatagramChannel(final ChannelFactory factory,
             final ChannelPipeline pipeline, final ChannelSink sink,
-            final NioDatagramWorker worker, InternetProtocolFamily family) {
+            final AbstractNioWorker worker, InternetProtocolFamily family) {
```

Comment:
NioDatagramChannel is used as both the server channel and the client channel. If we make NioServerDatagramChannel, then it can use boss then NioDatagramChannel can use NioWorker.